pkgs/build-support/docker/default.nix

```diff
@@ -125,6 +125,7 @@ rec {
   # And we cannot untar it, because then we cannot preserve permissions etc.
   tarsum = runCommand "tarsum" {
     nativeBuildInputs = [ go ];
+    disallowedReferences = [ go ];
```

Comment:
Instance of https://github.com/NixOS/nixpkgs/issues/132637 (no action required here now)